lib/tizen_sdk.dart

```diff
@@ -18,12 +18,22 @@ TizenSdk? get tizenSdk => context.get<TizenSdk>();
 
 File? get dotnetCli => globals.os.which('dotnet');
 
+/// The type of Tizen SDK installation.
+enum TizenSdkType {
+  /// VS code Tizen extension based SDK.
+  extension,
+
+  /// Tizen-Studio based SDK.
```

Comment:
As you know, Tizen-studio will soon be deprecated. Therefore, it seems necessary to use the Tizen CLI instead. Otherwise, there also is a way to maintain 3 options (vscode extension, tizen cli, tizen-studio).

Reply:
Changed `studio` to `cli`. cli is more appropriate.